main.go

```diff
@@ -34,6 +34,7 @@ import (
 
 const (
 	tdURI        = "trafficdirector.googleapis.com:443"
+	c2pTdURI     = "dns:///directpath-pa.googleapis.com"
```

Comment:
`c2pURI` for consistency with `c2pAuthority`?

Reply:
I was talking to Easwar about something similar (in the context of another PR). This generator script is actually kinda long. Making URIs string constants are actually a bit annoying since it forces us to scroll back and forth for just this one value. And most of these consts only have a single usage. Seems like you've had a similar discussion with Easwar about this. 

I've actually nixed the consts and used the strings as is. @ejona86 lmk what you think about it?